src/promptflow/promptflow/_utils/exception_utils.py

```diff
@@ -107,14 +106,14 @@ def get_user_execution_error_info(self):
         return user_execution_error_info
 
     def to_dict(self):
-        from promptflow._utils.utils import get_runtime_version
```

Comment:
Remove `get_runtime_version` from [utils.py](https://github.com/microsoft/promptflow/blob/6fba739b19f47936d22572e5a2bd659858e9b5de/src/promptflow/promptflow/_utils/utils.py#L173)?

Reply:
No. I haven't committed this change. Wanna commit together?

Reply:
separate commit is OK too, just a reminder. Seems it only usage is in `exception_utils.py`?

Reply:
Yes. Just removed.